tools/clang/test/HLSLFileCheck/validation/callgraph/barrier-group-in-nested-fn-vs-lib68.hlsl

```diff
@@ -0,0 +1,56 @@
+// RUN: %dxc -T lib_6_8 -Vd %s | %D3DReflect %s | %FileCheck %s -check-prefixes=RDAT
+
+// RDAT: FunctionTable[{{.*}}] = {
+
+RWBuffer<uint> Buf : register(u0);
+
+// RDAT-LABEL:   UnmangledName: "write_value"
+// RDAT:   FeatureInfo1: 0
+// RDAT:   FeatureInfo2: 0
+// RDAT:   ShaderStageFlag: (Pixel | Vertex | Geometry | Hull | Domain | Compute | Library | RayGeneration | Intersection | AnyHit | ClosestHit | Miss | Callable | Mesh | Amplification | Node)
+// RDAT:   MinShaderTarget: 0x60060
+
+[noinline] export
+void write_value(uint value) {
+    Buf[value] = value;
+}
+
+// RDAT-LABEL:   UnmangledName: "barrier_group"
+// RDAT:   FeatureInfo1: 0
+// RDAT:   FeatureInfo2: (Opt_RequiresGroup)
+// RDAT:   ShaderStageFlag: (Compute | Library | Mesh | Amplification | Node)
+// RDAT:   MinShaderTarget: 0x60060
+
+[noinline] export
+void barrier_group() {
+    write_value(1);
+    Barrier(GROUP_SHARED_MEMORY, GROUP_SCOPE);
+    write_value(2);
+}
+
+// RDAT-LABEL:   UnmangledName: "intermediate"
+// RDAT:   FeatureInfo1: 0
+// RDAT:   FeatureInfo2: (Opt_RequiresGroup)
+// RDAT:   ShaderStageFlag: (Compute | Library | Mesh | Amplification | Node)
+// RDAT:   MinShaderTarget: 0x60060
+
+[noinline] export
+void intermediate() {
+    write_value(3);
+    barrier_group();
+    write_value(4);
+}
+
+// RDAT-LABEL:   UnmangledName: "main"
+// RDAT:   FeatureInfo1: 0
+// RDAT:   FeatureInfo2: (Opt_RequiresGroup)
+// ShaderStageFlag indicates no compatible entry type after masking for vertex.
+// RDAT:   ShaderStageFlag: 0
+// MinShaderTarget still indicates vertex shader.
+// RDAT:   MinShaderTarget: 0x10060
+
+//[numthreads(8, 8, 1)]
```

Comment:
Intended inclusion?
```suggestion
```

Reply:
Yeah, I had been using and modifying a file to generate .ll before copying to here and forgot to remove the commented attribute.